src/data/pokemon-evolutions.ts

```diff
@@ -88,21 +88,88 @@ export class SpeciesEvolutionCondition {
   }
 }
 
-// TODO: Break this up into a MaleCondition and FemaleCondition?
-export class GenderEvolutionCondition extends SpeciesEvolutionCondition {
-  constructor(requiredGender: Gender) {
-    super((p) => p.gender === requiredGender);
-    this.description = "requires gender";
+/**
+ * For Pokemon that require being male, including
+ * Gallade
+ * Mothim
+ * Basculegion
+ * Oinkalogne
+ * Meowstic
+ *
+ * Custom:
+ * Glalie
+ * Huntail
```

Comment:
```suggestion
 * For Pokemon that require being male, including:
 * ```
 * Gallade
 * Mothim
 * Basculegion
 * Oinkalogne
 * Meowstic
 * ```
 *
 * Custom:
 * ```
 * Glalie
 * Huntail
 * ```
```